.github/workflows/cloud-tpu-ci-nightly.yml

```diff
@@ -56,6 +59,16 @@ jobs:
               -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
             pip install requests
 
+          elif [ "${{ matrix.jaxlib-version }}" == "nightly+oldest_supported_libtpu" ]; then
+          env:
```

Comment:
I don't think you can set an env var this way in the middle of the bash commands to run? Also this needs to be set for the test commands as well. You should just use a conditional in the regular env: block of the install + test steps (https://docs.github.com/en/actions/learn-github-actions/expressions#example)